tracing.go

```diff
@@ -146,6 +147,10 @@ type ClientConfig struct {
 	Exporter       exporterType `json:"type"`
 	Endpoint       string       `json:"url"`
 	Insecure       bool         `json:"insecure"`
+	TLSServerName  string       `json:"tls_server_name"`
+	TLSCertFile    string       `json:"tls_cert_file"`
+	TLSKeyFile     string       `json:"tls_key_file"`
+	TLSCAFile      string       `json:"tls_ca_file"`
```

Comment:
WDYT of directly vendoring the otel TLS config block?

Reply:
Ah yes, good idea.